descqa/shear_test.py

```diff
@@ -206,26 +211,26 @@ def get_catalog_data(gc, quantities, filters=None):
 
     # define theory from within this class
 
-    def post_process_plot(self, ax):
+    def post_process_plot(self, ax, fig):
         '''
         Post-processing routines on plot
         '''
-        zmeans = np.linspace(self.zlo, self.zhi, self.ntomo+2)[1:-1]
+        #zmeans = np.linspace(self.zlo, self.zhi, self.ntomo+2)[1:-1]
```

Comment:
Remove unused code instead of just commenting it out. If it must be kept, provide a reason inline.
```suggestion
```

Reply:
removed